code/modules/research/circuitprinter.dm

```diff
@@ -18,120 +18,125 @@ using metal and glass, it uses glass and reagents (usually sulfuric acis).
 	RefreshParts()
-		var/T = 0
-		for(var/obj/item/reagent_container/glass/G in component_parts)
-			T += G.reagents.maximum_volume
-		create_reagents(T, REFILLABLE)
-		T = 0
-		for(var/obj/item/stock_parts/matter_bin/M in component_parts)
-			T += M.rating
-		max_material_amount = T * 75000.0
 
-	proc/TotalMaterials()
-		return g_amount + gold_amount + diamond_amount + uranium_amount
 
-	attackby(var/obj/item/O as obj, var/mob/user as mob)
-		if (shocked)
-			shock(user,50)
-		if (isscrewdriver(O))
-			if (!opened)
-				opened = 1
-				if(linked_console)
-					linked_console.linked_imprinter = null
-					linked_console = null
-				icon_state = "circuit_imprinter_t"
-				to_chat(user, "You open the maintenance hatch of [src].")
-			else
-				opened = 0
-				icon_state = "circuit_imprinter"
-				to_chat(user, "You close the maintenance hatch of [src].")
-			return
-		if (opened)
-			if(iscrowbar(O))
-				playsound(src.loc, 'sound/items/Crowbar.ogg', 25, 1)
-				var/obj/machinery/constructable_frame/machine_frame/M = new /obj/machinery/constructable_frame/machine_frame(src.loc)
-				M.state = 2
-				M.icon_state = "box_1"
-				for(var/obj/I in component_parts)
-					if(istype(I, /obj/item/reagent_container/glass/beaker))
-						reagents.trans_to(I, reagents.total_volume)
-					if(I.reliability != 100 && crit_fail)
-						I.crit_fail = 1
-					I.loc = src.loc
-				if(g_amount >= 3750)
-					var/obj/item/stack/sheet/glass/G = new /obj/item/stack/sheet/glass(src.loc)
-					G.amount = round(g_amount / 3750)
-				if(gold_amount >= 2000)
-					var/obj/item/stack/sheet/mineral/gold/G = new /obj/item/stack/sheet/mineral/gold(src.loc)
-					G.amount = round(gold_amount / 2000)
-				if(diamond_amount >= 2000)
-					var/obj/item/stack/sheet/mineral/diamond/G = new /obj/item/stack/sheet/mineral/diamond(src.loc)
-					G.amount = round(diamond_amount / 2000)
-				if(uranium_amount >= 2000)
-					var/obj/item/stack/sheet/mineral/uranium/G = new /obj/item/stack/sheet/mineral/uranium(src.loc)
-					G.amount = round(uranium_amount / 2000)
-				qdel(src)
-				return 1
-			else
-				to_chat(user, "<span class='warning'>You can't load the [src.name] while it's opened.</span>")
-				return 1
-		if (disabled)
-			to_chat(user, "\The [name] appears to not be working!")
-			return
-		if (!linked_console)
-			to_chat(user, "\The [name] must be linked to an R&D console first!")
-			return 1
-		if (O.is_open_container())
-			return 0
-		if (!istype(O, /obj/item/stack/sheet/glass) && !istype(O, /obj/item/stack/sheet/mineral/gold) && !istype(O, /obj/item/stack/sheet/mineral/diamond) && !istype(O, /obj/item/stack/sheet/mineral/uranium))
-			to_chat(user, "<span class='warning'>You cannot insert this item into the [name]!</span>")
-			return 1
-		if (machine_stat)
-			return 1
-		if (busy)
-			to_chat(user, "<span class='warning'>The [name] is busy. Please wait for completion of previous operation.</span>")
+/obj/machinery/r_n_d/circuit_imprinter/RefreshParts()
```

Comment:
don't need to do this just delete everything after initialize, the code has already been butchered to not work.

Reply:
If I do that I'll have to mess with `/obj/machinery/computer/NTresearch` and other files that make use of these procs.
Removing this code is fine and dandy, but out of scope.